daemon/daemon_main.go

```diff
@@ -1102,7 +1102,7 @@ func runDaemon() {
 		if err != nil {
 			log.WithError(err).WithField("device", option.Config.FlannelMasterDevice).Fatal("Unable to set internal IPv4")
 		}
-		// d.attachExistingInfraContainers()
+		d.attachExistingInfraContainers()
```

Comment:
What will happen when this code runs and the Cilium DaemonSet is not running with `hostPID: true`?

Reply:
Currently yes, I'll think on something to avoid this.